test_1:
  planning_all:
    time:

  planning_1:
    time:
    resources:
      cpu: [move_group, firefox]
      mem: [move_group, rostest]

  planning_2:
    time:
    resources:
      cpu: [move_group, firefox]
      mem: [move_group, firefox]
      io: [move_group, firefox]
      network: [move_group, firefox]

  planning_3:
    time:
    resources:
      cpu: [move_group, phpstorm]
      mem: [move_group, phpstorm]

  execution_all:
    time:
    path_length:
      - [base_link, gripper_right_grasp_link]
      - [arm_right_3_link, gripper_right_grasp_link]

  execution_1:
    time:
    resources:
      cpu: [phpstorm]
      mem: [phpstorm]
      io: [move_group, firefox]
      network: [move_group, firefox]

  execution_2:
    time:
    path_length:
      - [base_link, gripper_right_grasp_link]
      - [base_link, arm_right_3_link]
    resources:
      cpu: [move_group, rostest]
      mem: [move_group, rostest]

  execution_3:
    time:
    resources:
      cpu: [rostest]

test_2:
  execution_1:
    time:
    path_length:
      - [base_link, gripper_right_grasp_link]
      - [arm_right_2_link, gripper_right_grasp_link]

  execution_2:
    time:
    path_length:
      - [base_link, gripper_right_grasp_link]
      - [arm_right_1_link, gripper_right_grasp_link]

  execution_3:
    time:
    path_length:
      - [base_link, gripper_right_grasp_link]
      - [arm_right_4_link, gripper_right_grasp_link]
